doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -86,7 +83,7 @@ of an environment variable, one makes a call like this::
     julia> bytestring(path)
     "/bin/bash"
 
-Note that the argument type tuple must be written as ``(Ptr{UInt8},)``,
```

Comment:
Actually, in 0.4 it should be written as `(Cstring,)` since it is a pointer to NUL-terminated bytestring.